set.mm

```diff
@@ -147154,6 +147148,405 @@ of an open range of nonnegative integers (of length equal to the length of
+    ZIJEZKZCALJGHZCAUIMNJGHZOJAUIARJZOJZUIUKULAUMUNOBCAPABCQUAUJASEUISEUOUIUBUH
+    UJAAUIUCTUHUIBCUDTAUIUEUFUG $.
+
+  $( The first symbol in a prefix of a word.  (Contributed by Alexander van der
+     Vekens, 16-Jun-2018.)  (Revised by AV, 3-May-2020.) $)
+  pfxfv0 $p |- ( ( W e. Word V /\ L e. ( 1 ... ( # ` W ) ) )
+                  -> ( ( W prefix L ) ` 0 ) = ( W ` 0 ) ) $=
+    ( cword wcel c1 chash cfv cfz co wa cc0 cfzo cpfx wceq simpl fz1ssfz0 sseli
+    adantl cn elfznn lbfzo0 sylibr pfxfv syl3anc ) CBDEZAFCGHZIJZEZKZUFALUGIJZE
+    ZLLAMJEZLCANJHLCHOUFUIPUIULUFUHUKAUGQRSUJATEZUMUIUNUFAUGUASAUBUCLABCUDUE $.
+
+  $( A symbol in a word truncated by one symbol.  (Contributed by Alexander van
+     der Vekens, 16-Jun-2018.)  (Revised by AV, 3-May-2020.) $)
+  pfxtrcfv $p |- ( ( W e. Word V /\ W =/= (/)
+                      /\ I e. ( 0 ..^ ( ( # ` W ) - 1 ) ) )
+                    -> ( ( W prefix ( ( # ` W ) - 1 ) ) ` I ) = ( W ` I ) ) $=
+    ( cword wcel chash cfv c1 cmin co cc0 cfz c0 wne cfzo cpfx wceq w3a wa cfn
+    wrdfin 1elfz0hash sylan cn lennncl elfz1end sylib 3adant3 fz0fzdiffz0 pfxfv
+    jca syl syld3an2 ) CBDEZCFGZHIJZKUOLJZEZCMNZAKUPOJEZACUPPJGACGQUNUSUTRHUQEZ
+    UOHUOLJEZSZURUNUSVCUTUNUSSZVAVBUNCTEUSVABCUACUBUCVDUOUDEVBBCUEUOUFUGUKUHUOH
+    UOUIULAUPBCUJUM $.
+
+  $( The first symbol in a word truncated by one symbol.  (Contributed by
+     Alexander van der Vekens, 16-Jun-2018.)  (Revised by AV, 3-May-2020.) $)
+  pfxtrcfv0 $p |- ( ( W e. Word V /\ 2 <_ ( # ` W ) )
+                     -> ( ( W prefix ( ( # ` W ) - 1 ) ) ` 0 ) = ( W ` 0 ) ) $=
+    ( cword wcel c2 chash cfv cle wbr wa c0 wne cc0 c1 cmin co cfzo cpfx wceq
+    cz simpl wrdlenge2n0 cn cuz 2z a1i lencl nn0zd adantr simpr eluz2 syl3anbrc
+    uz2m1nn syl lbfzo0 sylibr pfxtrcfv syl3anc ) BACDZEBFGZHIZJZUSBKLMMUTNOPZQP
+    DZMBVCRPGMBGSUSVAUAABUBVBVCUCDZVDVBUTEUDGDZVEVBETDZUTTDZVAVFVGVBUEUFUSVHVAU
+    SUTABUGUHUIUSVAUJEUTUKULUTUMUNVCUOUPMABUQUR $.
+
+  $( The last symbol in a (not empty) prefix of a word.  (Contributed by
+     Alexander van der Vekens, 24-Jun-2018.)  (Revised by AV, 3-May-2020.) $)
+  pfxfvlsw $p |- ( ( W e. Word V /\ L e. ( 1 ... ( # ` W ) ) )
+                    -> ( lastS ` ( W prefix L ) ) = ( W ` ( L - 1 ) ) ) $=
+    ( cword wcel c1 chash cfv cfz co wa cpfx clsw cmin wceq pfxcl adantr adantl
+    syl cc0 lsw fz1ssfz0 sseli pfxlen sylan2 fvoveq1d cfzo simpl elfznn fzo0end
+    cn pfxfv syl3anc 3eqtrd ) CBDZEZAFCGHZIJZEZKZCALJZMHZVAGHZFNJVAHZAFNJZVAHZV
+    ECHZUTVAUOEZVBVDOUPVHUSBCAPQVAUOUASUTVCAFVANUSUPATUQIJZEZVCAOURVIAUQUBUCZBC
+    AUDUEUFUTUPVJVETAUGJEZVFVGOUPUSUHUSVJUPVKRUSVLUPUSAUKEVLAUQUIAUJSRVEABCULUM
+    UN $.
+
+  ${
+    $d M i $.  $d N i $.  $d U i $.  $d V i $.  $d W i $.
+    $( The prefixes of two words are equal iff they have the same length and
+       the same symbols at each position.  (Contributed by Alexander van der
+       Vekens, 7-Aug-2018.)  (Revised by AV, 4-May-2020.) $)
```

Comment:
That's true for any word. Is it useful to have it for prefixes ?

Reply:
This theorem (or its predecessor ~ swrdeqOLD) is used 3 times, therefore I would like to keep it.